spec.html

```diff
@@ -14348,7 +14422,26 @@ <h2>Syntax</h2>
         <p>&lt;LF&gt; and &lt;CR&gt; cannot appear in a string literal, except as part of a |LineContinuation| to produce the empty code points sequence. The proper way to include either in the String value of a string literal is to use an escape sequence such as `\\n` or `\\u000A`.</p>
       </emu-note>
 
-      <emu-clause id="sec-static-semantics-sv" oldids="sec-string-literals-static-semantics-stringvalue" type="sdo" aoid="SV">
+      <emu-clause id="sec-string-literals-early-errors">
+        <h1>Static Semantics: Early Errors</h1>
+        <emu-grammar>
+          EscapeSequence :: LegacyOctalEscapeSequence
+
+          EscapeSequence :: NonOctalDecimalEscapeSequence
+        </emu-grammar>
+        <ul>
+          <li>It is a Syntax Error if the source code matching this production is strict mode code.</li>
+        </ul>
+        <emu-note>In non-strict code, this syntax is allowed, but deprecated.</emu-note>
```

Comment:
I think we need to use a word other than "deprecated" here. I think "discouraged" or "considered legacy" (if we want to add a description of the committee's definition of "legacy") would be better.

Reply:
We will wait for #2125 to land and then use the term "legacy" here, which will link to its definition.